README.md

```diff
@@ -51,14 +51,72 @@ The only prerequisites for running it are [Nextflow](https://www.nextflow.io/) a
 
 <a name="install"></a>
 
-## Install and dependencies
+## Install and downloading dependencies
 
 To get a copy of the Mobilome Annotation Pipeline, clone this repo by:
 
 ```bash
 $ git clone https://github.com/EBI-Metagenomics/mobilome-annotation-pipeline.git
 ```
 
+The first time you run the pipeline you will need to set up the following databases:
+
+1. Download and index the database for amrfinder plus
+```bash
+wget -r -nH --cut-dirs=5 ftp://ftp.ncbi.nlm.nih.gov/pathogen/Antimicrobial_resistance/AMRFinderPlus/database/3.12/2024-01-31.1/
+singularity pull ncbi-amrfinderplus_3.12.8.sif docker://depot.galaxyproject.org/singularity/ncbi-amrfinderplus:3.12.8--h283d18e_0
+singularity exec -B $(pwd):/data ncbi-amrfinderplus_3.12.8.sif amrfinder_index 2024-01-31.1
+```
+
+2. Download and extract the geNomad database
+```bash
+wget https://zenodo.org/records/14886553/files/genomad_db_v1.9.tar.gz?download=1
+tar -xvf genomad_db_v1.9.tar.gz
+```
+
+3. Download and extract the databases to run ICEfinder2-lite
+```bash
+wget https://ftp.ebi.ac.uk/pub/databases/metagenomics/pipelines/tool-dbs/icefinder2lite/icf2_dbs.tar.gz
+tar -xvf icf2_dbs.tar.gz
+```
+
+4. Uniprotkb-sp download, decompress, formatting and indexing for blastp. This is also for ICEfinder2-lite
+```bash
+wget https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz
+gzip -d uniprot.dat.gz
+singularity pull prokka-uniprot-1.14.5.sif docker://quay.io/microbiome-informatics/prokka-uniprot:1.14.5
+singularity exec -B $(pwd):/data prokka-uniprot-1.14.5.sif prokka-uniprot_to_fasta_db uniprot_sprot.dat > prokka_uniprot_sprot.fasta
+singularity exec -B $(pwd):/data prokka-uniprot-1.14.5.sif makeblastdb -dbtype prot -in prokka_uniprot_sprot.fasta
+rm uniprot_sprot.dat prokka_uniprot_sprot.fasta
+```
+
+Once donwloading is complete, you can move the files to any suitable location. We recomment to create a config file with the following paths and pass it to the pipeline during execution using `-c my_paths.config`
```

Comment:
I would instruct users to use the corresponding --flags instead and mention the config approach.

```
--amrfinder_plus_db PATH...
```

Reply:
Ah! Instead of creating the config file? I will add that option as well to the README file